vmm/task/src/main.rs

```diff
@@ -142,16 +144,10 @@ lazy_static! {
     ]);
 }
 
-async fn initialize() -> anyhow::Result<()> {
-    early_init_call().await?;
```

Comment:
Please do not change `early_init_call()`

Reply:
Fixed.